scripts/zones/Inner_Horutoto_Ruins/mobs/Magicked_Bones.lua

```diff
@@ -0,0 +1,27 @@
+require("scripts/globals/utils")
+-----------------------------------
+-- Area: Inner Horutoto Ruins
+--  Mob: Magicked Bones
+-----------------------------------
+local ID = zones[xi.zone.INNER_HORUTOTO_RUINS]
+-----------------------------------
+local entity = {}
+
+local weaverId = 17563669
+local thugId = 17563668
+
+local daggerId = 17563671
+local clubId = 17563670
```

Comment:
These are redundant, they should all be references in the ID.mob table

Reply:
That is an option, I decided to opt for this rather than using the entries in IDs.lua that are already there since I didn't want to use a loop just to get a single value since the entries in ID.mobs are stored as tables and generically accessing them via pairs seemed like extra unnecessary overhead.

Reply:
yep, the first column is the lua file that gets read:

![image](https://github.com/LandSandBoat/server/assets/131182600/e9cd40a1-4cd8-414a-9127-dfe4a67d0bfc)

Scratch renaming it though just put an pair of if checks in the `onMobInitialize` function imo.

Reply:
Mob IDs and any entity ID for that matter, shifts every every version update.

We can't. and wont, gofile by file in the repository updating the IDs. It just isnt feasable. Hence why we have a zone file with mob IDs in it.

And now, we use a new pattern of `MOB_NAME = GetFirstID('Mob_Database_Name')` which allows us to just dont need to update them every version update,

This will only work from the zone.lua files.

Reply:
Ah alright so that is the new pattern. This only works specifically in Zone.lua, or did you mean zone wide .lua files, aka IDs.lua AND Zone.lua? Since all of the IDs so far have been in IDs, that was where I would expect GetFirstID to work. Glad you sent this, I was just in the middle of making a new commit but I can tweak my ID changes again to match that pattern better if it works in IDs.lua

Reply:
well, i meant in the IDs.lua files inside zone folders. Sorry, I wasnt clear.

Reply:
Alright cool, so I will switch to using that then. I am curious how this pattern actually works though, since if I had to guess not every placeholder would be the first id of their mobtype in a zone, would they?